.github/workflows/test-only.yml

```diff
@@ -87,15 +87,54 @@ jobs:
 
       - name: Test changedetection.io container starts+runs basically without error
         run: |
-          docker run -p 5556:5000 -d test-changedetectionio
+          docker run --name test-changedetectionio -p 5556:5000  -d test-changedetectionio
           sleep 3
           # Should return 0 (no error) when grep finds it
           curl -s http://localhost:5556 |grep -q checkbox-uuid
           
           # and IPv6
           curl -s -g -6 "http://[::1]:5556"|grep -q checkbox-uuid
+          docker kill test-changedetectionio
+
+      - name: Test changedetection.io SIGTERM and SIGINT signal shutdown
+        run: |
+          
+          echo SIGINT Shutdown request test
+          docker run --name sig-test -d test-changedetectionio
+          sleep 20
+          echo ">>> Sending SIGINT to sig-test container"
+          docker kill --signal=SIGINT sig-test
+          sleep 10
+          # invert the check (it should be not 0/not running)
+          docker ps
+          docker logs sig-test
+          EXIT_INT=$(docker inspect --format '{{json .State.ExitCode}}' sig-test)
+          if [ ${EXIT_INT} != 0 ]
+          then
+            exit 1
+          fi
+          
+          # @todo - scan the container log to see the right "graceful shutdown" text exists 
+          docker rm sig-test
+          
+          echo SIGTERM Shutdown request test
+          docker run --name sig-test -d test-changedetectionio
+          sleep 20
+          echo ">>> Sending SIGTERM to sig-test container"
+          docker kill --signal=SIGTERM sig-test
+          sleep 10
+          # invert the check (it should be not 0/not running)
+          docker ps
+          docker logs sig-test
+          EXIT_TERM=$(docker inspect --format '{{json .State.ExitCode}}' sig-test)
+          if [ ${EXIT_TERM} != 0 ]
+          then
+            exit 1
+          fi
           
+          # @todo - scan the container log to see the right "graceful shutdown" text exists           
```

Comment:
see above content, these two tests add a full minute and are error prone

Reply:
For the todo, How about check the log with `docker logs -f xpath_for_infra 2> /dev/null | grep Gracefully`?
Without `2> /dev/null`,
```
[shelf@localhost git_worktree_changedetection]$ podman logs -f xpath_for_infra  | grep Gracefully
(1) wsgi starting up on http://0.0.0.0:5000
Gracefully exiting
wsgi exiting
(1) wsgi exited, is_accepting=True
```
```
[shelf@localhost git_worktree_changedetection]$ podman logs -f xpath_for_infra 2> /dev/null | grep Gracefully
Gracefully exiting
```

Reply:
I will add a graceful exited print statement with a global variable later. So only print the 'graceful exited' when triggered by the signals. I wanted the wsgi.close method, but the lib doesn't have such method.

Reply:
I added test! Thanks!